bin/acpp

```diff
@@ -346,6 +346,9 @@ class acpp_config:
       'is-dryrun': option("--acpp-dryrun", "ACPP_DRYRUN", "default-is-dryrun",
 """  If set, only shows compilation commands that would be executed,
   but does not actually execute it. """),
+      'is-dryrun-noplugin': option("--acpp-dryrun-noplugin", "ACPP_DRYRUN_NOPLUGIN", "default-is-dryrun-noplugin",
```

Comment:
Is this still an appropriate name? It looks like what we are doing is more like a give-me-all-standard-flags type of thing.

Reply:
agreed, i renamed it to `--acpp-dryrun-only-std-flags`